### PR TITLE
feat: Added responsiveness

### PR DESCRIPTION
**Pull request by BalajiHarinath**
I have made the app responsive. Please review.

[Live Link](https://steady-youtiao-594985.netlify.app/)

### CODE REVIEW

**Review by anik31 — APPROVED** [review]
Code is clean & the media queries you have written seems fine. It's good that you've made it responsive.

suggestion - try to give preview links for ui reviews

File: src/components/SidebarNav/SidebarNav.css [hunk 1]
```diff
@@ -15,3 +15,18 @@
...
 }
+
+/*Media Queries*/
+@media screen and (max-width: 850px) {
+  .container-sidebar {
+    width: fit-content;
+  }
+
+  .icon-sidebar-small-screen {
+    padding: 0 0.3rem;
```
Comment (sainath7878) [inline src/components/SidebarNav/SidebarNav.css:26]: suggestion: You can use px if the value is less than 10px.

**Review by sainath7878 — APPROVED** [review]
Everything looks good. Nice work and great UI!

File: src/components/SidebarNav/SidebarNav.css (hunk 1 above)
Comment (BalajiHarinath) [reply src/components/SidebarNav/SidebarNav.css:26]: will change it.

**Pull request merged**